### DISCUSSION TITLE
Link peaks to genes for non-multiome data

### DISCUSSION THREAD

**Discussion by cadyyuheng** [stuart-lab/signac#466] [Q&A] [answered]
Hi,

In the online tutorial, it shows how we could link peak to genes for 10x multiomic and snare-seq. I was wondering if it's possible to use linkpeak function on separate 10X scRNA-seq and scATAC-seq objects of the same population.

Thanks!

**Comment by timoast** [comment, marked as answer]
If you're interested in doing this and have paired scRNA and scATAC (same tissue but different cells), I'd suggest imputing gene expression values in the scATAC-seq dataset by multimodal integration. See the Seurat vignette here for an example of how to impute RNA in scATAC-seq data: https://satijalab.org/seurat/articles/atacseq_integration_vignette.html#co-embedding-scrna-seq-and-scatac-seq-datasets

Once you have the imputed RNA expression values in the scATAC-seq dataset, you can use the LinkPeaks function in Signac as shown in the Signac vignette: https://satijalab.org/signac/articles/pbmc_multiomic.html#linking-peaks-to-genes

Just keep in mind that the links will have been computed from predicted, rather than measured, gene expression, and so may not be as accurate as those determined by actual multiome data.

**Reply by cadyyuheng** [reply]
Thanks @timoast ! I have imputed scATAC, and as you said, I noticed the imputation results do not perfectly reflect my scRNA data. Nevertheless, I will give it a try with LinkPeaks.

I really appreciate your help!
